docs/data/ResearchStudy-sd-001.json

```diff
@@ -1,6 +1,6 @@
 {
     "resourceType":"ResearchStudy",
-    "id": "rs-001",
+    "id": "sd-001",
     "meta": {
         "profile": ["http://fhir.kids-first.io/StructureDefinition/study"]
```

Comment:
The profile doesn't need to be `"http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study"` since this is a valid example?